src/module.c

```diff
@@ -1038,6 +1038,10 @@ initpycurl(void)
 #endif
 #if LIBCURL_VERSION_NUM >= MAKE_LIBCURL_VERSION(7, 47, 0)
     insint_c(d, "CURL_HTTP_VERSION_2TLS", CURL_HTTP_VERSION_2TLS);
+#endif
+#if LIBCURL_VERSION_NUM >= MAKE_LIBCURL_VERSION(7, 49, 0)
+    insint_c(d, "CURL_HTTP_VERSION_2_PRIOR_KNOWLEDGE", CURL_HTTP_VERSION_2_PRIOR_KNOWLEDGE);
+	insint_c(d, "CURLOPT_TCP_FASTOPEN", CURLOPT_TCP_FASTOPEN);
```

Comment:
Thanks, the only change needed is the Python symbol should be `TCP_FASTOPEN` (without the `CURLOPT` prefix).